lib/Crypto/DecryptAll.php

```diff
@@ -90,55 +100,98 @@ public function __construct(
 	 * @throws LoginException
 	 */
 	public function prepare(InputInterface $input, OutputInterface $output, $user) {
-
-		$question = new Question('Please enter the recovery key password: ');
-
+		// We need to setup the module with everything it needs in order to decrypt user data
+		// This could be using master key or user key encryption
+		// and could also be for a single user or multiple users
+
+		//Try to stop decrypting process if password method is used for decrypting all users.
+		if (($input->getArgument('user') === '') && ($input->getOption('method') === 'password')) {
```

Comment:
why check the user argument in here when it's already passed as `$user` ?

Reply:
Thought of adding an extra precaution.

Reply:
I have updated the comments in the latest version of the PR. Let me know if the description looks ok?